src/roman_datamodels/stnode.py

```diff
@@ -252,30 +261,66 @@ def get_schema(self):
             schema_uri, self.ctx, False, False)
         return schema
 
+
 class TaggedListNode(LNode):
 
     @property
     def tag(self):
         return self._tag
 
 
+_DATAMODELS_MANIFEST_PATH = importlib_resources.files(rad.resources) / "manifests" / "datamodels-1.0.yaml"
+_DATAMODELS_MANIFEST = yaml.safe_load(_DATAMODELS_MANIFEST_PATH.read_bytes())
+_OBJECT_NODE_CLASS_NAME_OVERRIDES = {
+    "tag:stsci.edu:datamodels/roman/reference_files/flat-1.0.0": "FlatRef",
```

Comment:
There are going to be a lot of these. Should there be more generic handling of reference file tags?

Reply:
That's a good idea, shall I just add `Ref` to the ends of their names?

Reply:
Yes.

Reply:
This is done